### DISCUSSION TITLE
Vector2 Pane Item

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#111] [Feedback]
I think that adding a Vector2 Item to the Pane is better than using two Number Pane Items

![B0233A51-C0B0-4677-A1BF-A45FA34142E5](https://user-images.githubusercontent.com/47607185/235163967-b51cfcf1-4a06-4807-a5de-9b351316429f.png)

**Comment by minecraftchloe** [comment]
Please tell us more about the scenarios for Vec2 vs Vec3! It'll help with understanding your thought process. Do you want both supported?

**Reply by xKingDark** [reply]
Having both would be useful, the reason I want Vec2 is because the player's rotation uses Vec2 and having to use two Number Pane Items to change it isn't great

**Reply by dacowan** [reply]
Vec2 UI components are already on my list :)

**Reply by xKingDark** [reply]
👀 Thank you!
